cmd/cel-key/main.go

```diff
@@ -31,26 +29,6 @@ var initClientCtx = client.Context{}.
 
 var rootCmd = keys.Commands("~")
 
-func init() {
```

Comment:
The additional "convenience" provided by allowing users to pass directory flags to cel-key doesn't justify how buggy and inconsistent that was, so instead we should just ask users to pass the path to the directory where their keys are if they want to work with the keys. We should document the default paths well.